.github/workflows/ci-base-tests-linux.yml

```diff
@@ -47,4 +47,5 @@ jobs:
             --ignore=./smarts/core/tests/test_env_frame_rate.py \
             --ignore=./smarts/env/tests/test_benchmark.py \
             --ignore=./examples/tests/test_learning.py \
+            --ignore=./smarts/core/tests/test_parallel_sensors.py \
```

Comment:
Why was the parallel sensors test ignored?

Reply:
The parallel test causes problems on the CI because it uses a lot of CPU's and the CI only has 2 (which are shared among many tests.)